src/kaocha/runner.clj

```diff
@@ -148,29 +148,12 @@
 
   (binding [spec/*explain-out* expound/printer]
     (let [{{:keys [config-file plugin profile]} :options} (cli/parse-opts args cli-options)
-          config                                          (-> config-file
-                                                              (or "tests.edn")
-                                                              (config/load-config (if profile
-                                                                                    {:profile profile}
-                                                                                    {})))
-          _check_config_file                              (when (not (. (File. (or config-file "tests.edn")) exists))
```

Comment:
I see now where the underscores come from. And I guess the `.` syntax was already here. These sections are really only here for side effects I believe, hence why the names start with an `_`.